app/src/main/java/org/p2p/wallet/moonpay/ui/transaction/SellTransactionDetailsBottomSheet.kt

```diff
@@ -214,6 +215,21 @@ class SellTransactionDetailsBottomSheet :
         binding.layoutDetails.buttonRemoveOrCancel.isLoadingState = isLoading
     }
 
+    override fun navigateToSendScreen(
+        tokenToSend: Token.Active,
+        sendAmount: BigDecimal,
+        receiverAddress: String
```

Comment:
Maybe you will pass `SearchResult.AddressFound` here?

Reply:
but it does

Reply:
val recipient = SearchResult.AddressFound(AddressState(receiverAddress)) 
just a line down from this code